pkg/controllers/consistency/metrics.go

```diff
@@ -26,14 +26,14 @@ func init() {
 }
 
 const (
-	checkLabel = "check"
-	subsystem  = "consistency"
+	checkLabel           = "check"
+	consistencySubsystem = "consistency"
```

Comment:
Why do you prefer the stutter with package name.

Reply:
It's primarily for the metrics_gen in aws/karpenter. We have a [mapping pre-defined when the namespace is a variable name](https://github.com/aws/karpenter/blob/main/hack/docs/metrics_gen_docs.go#L259). If the name of the variable is just the name "subsystem", we don't know how to look up the name for the subsystem. I'd like to change this in our generation mechanism but this is the state of things right now.